apps/build.sh

```diff
@@ -17,7 +17,7 @@ HERE=$(dirname $(readlink -f $0))
 require_params FACTORY
 
 export DOCKER_BUILDKIT=1
-BUILDKIT_VERSION="${BUILDKIT_VERSION-v0.10.3}"
+BUILDKIT_VERSION="${BUILDKIT_VERSION-v0.13.1}"
```

Comment:
are we sure this change is needed. If so - can we update the commit message to explain why?

Also - Can we bump this without bumping the docker-dind-ci container image?

Reply:
No, I am not sure. Let me double check it.

Reply:
```
|  #1 DONE 5.5s
|  ERROR: multiple outputs currently unsupported by the current BuildKit daemon, please upgrade to version v0.13+ or use a single output
```